wdl2cwl/main.py

```diff
@@ -409,13 +413,40 @@ def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:  # type: ignore
             return f"{left_operand} !== {right_operand}"
         elif function_name == "read_string":
             only_arg = arguments[0]
-            only_arg = self.get_expr(only_arg)  # type: ignore
-            return only_arg  # type: ignore
+            only_arg_value = self.get_expr(only_arg)
+            return only_arg_value
         elif function_name == "glob":
             only_arg = arguments[0]
             glob = self.get_expr(only_arg)
             return glob
-
+        elif function_name == "ceil":
+            only_arg = arguments[0]
+            only_arg_expr = self.get_expr(only_arg)
+            return f"Math.ceil({only_arg_expr}) "
+        elif function_name == "_mul":
+            left_operand, right_operand = arguments
+            if left_operand.literal:
+                left_operand_value = left_operand.literal.value
+            else:
+                raise ValueError(
+                    f"Left operand {left_operand} of the mul function has no literal"
+                )
+            if right_operand.literal:
+                raise ValueError(
+                    "Mul function with literal right_operand, not yet handled."
+                )
+            else:
+                right_operand_value = self.get_expr(right_operand)
+            return f"{left_operand_value}*({right_operand_value}) "
+        elif function_name == "size":
+            left_operand, right_operand = arguments
+            input_file_name = self.get_expr_name(left_operand.expr)  # type:ignore
+            size_unit = self.get_expr(right_operand)[1:-1]
+            if size_unit == "Gi":
+                unit_value = 3
+            else:
+                raise ValueError(f"Size unit {size_unit} not handle yet")
```

Comment:
I get this, but it would be nicer to handle all the WDL units now https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#units-of-storage